tests/by-util/test_cksum.rs

```diff
@@ -663,3 +663,195 @@ fn test_conflicting_options() {
         )
         .code_is(1);
 }
+
+#[test]
+fn test_check_algo_err() {
+    let scene = TestScenario::new(util_name!());
+
+    let at = &scene.fixtures;
+
+    at.touch("f");
+
+    scene
+        .ucmd()
+        .arg("--a")
```

Comment:
While this works fine, for consistency reasons I would use `-a` (or `--algorithm`).
```suggestion
        .arg("-a")
```